### ISSUE TITLE
Jest test failed: SyntaxError: Unexpected string

### ISSUE THREAD

**Issue by hxuanhung** [jaydenseric/apollo-upload-client#115]
I have jest's config as follow:
```
 "transform": {
      "^.+\\.(js|jsx|mjs)$": "<rootDir>/node_modules/babel-jest",
      "^.+\\.tsx?$": "<rootDir>/config/jest/typescriptTransform.js",
      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
      "^(?!.*\\.(js|jsx|mjs|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
    },
    "transformIgnorePatterns": [
      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|ts|tsx)$"
    ],
``` 
and when I run it gives me the following error: 
```
 /Users/hunghoang/workspace/admin-client/node_modules/apollo-upload-client/node_modules/extract-files/lib/index.mjs:1    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import 'core-js/modules/es6.function.name'                                                                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    SyntaxError: Unexpected string
```

if I try to remove `mjs` from `transformIgnorePatterns` in the config then no more error.

**Comment by jaydenseric** [comment]
Without deep diving into your issue, a few things to keep in mind...

This should be different in the next, not-yet-released version: https://github.com/jaydenseric/apollo-upload-client/blob/master/changelog.md#next.

The latest version of the extract-files dependency no longer uses the Babel runtime: https://github.com/jaydenseric/extract-files/blob/master/changelog.md#400.

**Comment by jaydenseric** [comment]
I just published [v9.0.0](https://github.com/jaydenseric/apollo-upload-client/releases/tag/v9.0.0), please let me know if there are any further issues 🙏
